.github/workflows/bow-php-action.yml

```diff
@@ -1,10 +1,6 @@
 name: Bow Framework CI
 
-on:
-  push:
-  pull_request:
-  schedule:
-    - cron: '0 0 * * 0'
+on: [ push, pull_request ]
```

Comment:
Listening every upcoming `pull` and `pull_request` trigger rather then using the schedule to run the CI.